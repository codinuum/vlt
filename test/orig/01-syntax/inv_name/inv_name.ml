let () = [%LOG [%DEBUG] "msg" [%NAME 0]]
