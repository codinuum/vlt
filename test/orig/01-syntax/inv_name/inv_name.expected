File "inv_name.ml", line 1, characters 37-38:
1 | let () = [%LOG [%DEBUG] "msg" [%NAME 0]]
                                         ^
Error: string expected
